packages/scaffold/src/client.ts

```diff
@@ -177,6 +178,10 @@ export class Web3ModalScaffold {
       AccountController.setAddressExplorerUrl(addressExplorerUrl)
     }
 
+  protected setMetadata(metadata: OptionsControllerState['metadata']) {
```

Comment:
Similar to comment above is this now needed? You have metadata available on options and can call `OptionsController.setMetadata(options.metadata)` inside `initControllers`